security-analysis/security-analysis-api/src/main/java/com/powsybl/security/LimitViolation.java

```diff
@@ -114,7 +114,7 @@ public LimitViolation(String subjectId, String subjectName, LimitViolationType l
      * @param value          The actual value of the physical value which triggered the detection of a violation.
      */
     public LimitViolation(String subjectId, LimitViolationType limitType, double limit, float limitReduction, double value) {
-        this(subjectId, null, limitType, limit, limitReduction, value);
+        this(subjectId, null, limitType, null, Integer.MAX_VALUE, limit, limitReduction, value, null);
     }
```

Comment:
sounds like an API breaking change, could this be avoided ?

Reply:
Not sure about what you mean by breaking change ?

Reply:
minor issue